docs/ballerina-by-example/examples/inbound-response-with-multiparts/inbound-response-with-multiparts.bal

```diff
@@ -0,0 +1,83 @@
+import ballerina.net.http;
```

Comment:
Missing license header

Reply:
we do not add license headers to ballerina by samples

Reply:
Oh ok! I didn't notice this is one of them.